lib/f.pl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
```

Comment:
I guess you didn't want this file in there :)

Reply:
oops